daphne_server/src/roles/aggregator.rs

```diff
@@ -155,7 +399,25 @@ impl DapReportInitializer for crate::App {
         part_batch_sel: &PartialBatchSelector,
         consumed_reports: Vec<EarlyReportStateConsumed>,
     ) -> Result<Vec<EarlyReportStateInitialized>, DapError> {
-        todo!()
+        tokio::task::spawn_blocking({
+            let vdaf_config = task_config.vdaf.clone();
+            let vdaf_verify_key = task_config.vdaf_verify_key.clone();
+            move || {
+                consumed_reports
+                    .into_par_iter()
+                    .map(|consumed_report| {
+                        EarlyReportStateInitialized::initialize(
+                            is_leader,
+                            &vdaf_verify_key,
+                            &vdaf_config,
+                            consumed_report,
+                        )
+                    })
+                    .collect::<Result<Vec<EarlyReportStateInitialized>, _>>()
+            }
+        })
+        .await
+        .map_err(|e| fatal_error!(err = ?e, "initialzing reports panicked"))?
```

Comment:
It didn't really panic, did it?
```suggestion
        .map_err(|e| fatal_error!(err = ?e, "encountered error while initializing reports"))?
```

Reply:
It did actually panic. This error happens because whatever was passed into `tokio::task::spawn_blocking` panicked. This panic is caught and returned here in the Err variant